src/main/java/by/Starleken/utils/EntityManagerUtils.java

```diff
@@ -0,0 +1,17 @@
+package by.Starleken.utils;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.EntityManagerFactory;
+import jakarta.persistence.Persistence;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EntityManagerUtils {
```

Comment:
`{...}Utils` classes should not be annotated as Beans (does not annotate with `@Component` / `@Service`)

`{...}Utils` classes should contain only static methods. Do not inject it to other beans

==============

But if you want this service to be a bean please move this class to `services` package and remove `Utils` suffix from its name. `EntityManagerUtils` -> `EntityManagerProvider`. And in this case you should keep the `@Component` annotation